conformant = "Pass"
output = """
generics_defaults.py:24:7 - error: "T" cannot appear after "DefaultStrT" in type parameter list because it has no default type (reportGeneralTypeIssues)
generics_defaults.py:52:16 - error: Too few type arguments provided for "AllTheDefaults"; expected 2 but received 1 (reportInvalidTypeArguments)
generics_defaults.py:107:35 - error: TypeVar default type must be a subtype of the bound type (reportGeneralTypeIssues)
generics_defaults.py:116:36 - error: TypeVar default type must be one of the constrained types (reportGeneralTypeIssues)
generics_defaults.py:144:7 - error: TypeVar "T5" has a default value and cannot follow TypeVarTuple "Ts" (reportGeneralTypeIssues)
generics_defaults.py:173:18 - error: Access to generic instance variable through class is ambiguous (reportGeneralTypeIssues)
"""
conformance_automated = "Fail"
errors_diff = """
Line 24: Unexpected errors ['generics_defaults.py:24:7 - error: "T" cannot appear after "DefaultStrT" in type parameter list because it has no default type (reportGeneralTypeIssues)']
Line 52: Unexpected errors ['generics_defaults.py:52:16 - error: Too few type arguments provided for "AllTheDefaults"; expected 2 but received 1 (reportInvalidTypeArguments)']
Line 107: Unexpected errors ['generics_defaults.py:107:35 - error: TypeVar default type must be a subtype of the bound type (reportGeneralTypeIssues)']
Line 116: Unexpected errors ['generics_defaults.py:116:36 - error: TypeVar default type must be one of the constrained types (reportGeneralTypeIssues)']
Line 144: Unexpected errors ['generics_defaults.py:144:7 - error: TypeVar "T5" has a default value and cannot follow TypeVarTuple "Ts" (reportGeneralTypeIssues)']
Line 173: Unexpected errors ['generics_defaults.py:173:18 - error: Access to generic instance variable through class is ambiguous (reportGeneralTypeIssues)']
"""
